webserial.js

```diff
@@ -1,9 +1,12 @@
-import {TimeoutError} from './error.js';
+'use strict';
+
+import { TimeoutError } from "./error";
```

Comment:
We don't need use strict. It's imported as a module so that's implied. You also lost the extension on the import.

```suggestion
import { TimeoutError } from "./error.js";
```

Reply:
Without this change it's not possible to test this PR from the embedded website. How have you been testing this?

Reply:
Sorry about that one. Not sure how that slipped in.
Yes, I have normally been testing the code, but I am switching between branches a lot as I'm integrating this code into a web based application for flashing ExpressLRS binaries.